interop-cats/shared/src/main/scala/zio/interop/Queue.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2017-2019 John A. De Goes and the ZIO Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package zio.interop
+
+import cats.effect.LiftIO
+import zio.{ Queue => ZioQueue, Runtime, UIO, ZEnv }
+
+object Queue {
+
+  /**
+   * @see [[ZioQueue.bounded]]
+   */
+  final def bounded[F[+_], R, A](capacity: Int)(implicit R: Runtime[ZEnv], F: LiftIO[F]): F[Queue[F, A]] =
```

Comment:
Isn't the `R` type parameter useless here?

Reply:
Great point!

Reply:
Saw it from the tests actually, `Queue.dropping[F, Any, Int](2)` can simply be `Queue.dropping[F, Int](2)`

Reply:
Yeah you are right 👌

Reply:
Fixed :)